ColorHighlighter.py

```diff
@@ -1241,6 +1242,37 @@ def get_vars(self, view):
                     if color_vars_file is not None:
                         color_vars_files.append(color_vars_file)
 
+                    # If not opened as a project
```

Comment:
This code should be put in a separate function `parse_dot_colorhighlighter(pdata, color_vars_files)`. I know the current code is by far not in a good state, but let's not make it any worse.